backend/apps/schemes/services/benefit_service.py

```
@@ -1,27 +1,60 @@
+        StringLengthRule("description", max_length=500, min_length=None),
+    ]
 
     # ---------------------------------------------------------------------
     # Basic CRUD Operations
     # ---------------------------------------------------------------------
 
-    @staticmethod
+    @classmethod
     @transaction.atomic
-    def benefit_create(*, benefit_data: dict, user=None):
+    def benefit_create(cls, *, benefit_data: dict, user=None):
```

Ruff (0.14.3)

Unused class method argument: `user`

(ARG003)